### ISSUE TITLE
Unable to start session due to QML bug

### ISSUE THREAD

**Issue by ToddAWalter** [moonlight-stream/moonlight-qt#1350]
**Describe the bug**
After successfully adding a host PIN, I'm unable to start the connection (this is on v6.0 L4T; prior version worked).  Two error messages appear for every click, below:

Qt Warning: QQmlComponent: Component is not ready
Qt Warning: qrc:/gui/main.qml:82:5: QML StackView: push: nothing to push

**Steps to reproduce**
Install moonlight-qt on your Jetson Nano and try to connect.

**Other Moonlight clients**
Connections to the host from my other clients works fine.

**Client PC details (please complete the following information)**
 - OS: Ubuntu Bionic Beaver (final image available for the Jetson Nano from nVidia)
 - Moonlight Version: bionic, 6.0.1-1 arm64
 - GPU: Nvidia
 - Linux package type (if applicable): deb

**Server PC details (please complete the following information)**
 - OS: Gentoo Linux
 - Sunshine or GeForce Experience version: sunshine 0.23.1
 - GPU: Quadro RTX 4000
 - GPU driver: 555.58.02

**Comment by deroli-us** [comment]
Same Bug over here for the last few days. Raspi 4 with retropie installed as client and Pop os für the server. All my other Devices work.

**Comment by elsw** [comment]
Same issue here.

**Client**
    OS: JETSON Nano running Ubuntu 18
    Moonlight Version: 6.0.1

**Server**
    OS: Windows 11
    Sunshine v0.23.1
    GPU: RTX 3060
    GPU driver: 536.99

**Comment by gabrielformiga** [comment]
Same problem, also tried streaming directly from terminal, it opened up a new window, but always black, works fine on other clients.

Client
OS: Nintendo Switch L4T Ubuntu 18.04
Moonlight Version: 6.0.1

**Comment by gabrielformiga** [comment]
That is the moonlight-qt fixed for L4T Bionic (tested on a nintendo switch), if someone else wants to try
[moonlight-qt-6.0.1-1-l4t-bionic.zip](https://github.com/user-attachments/files/16664223/moonlight-qt-6.0.1-1-l4t-bionic.zip)

**Comment by ToddAWalter** [comment]
> That is the moonlight-qt fixed for L4T Bionic (tested on a nintendo switch), if someone else wants to try [moonlight-qt-6.0.1-1-l4t-bionic.zip](https://github.com/user-attachments/files/16664223/moonlight-qt-6.0.1-1-l4t-bionic.zip)

Thanks.  This works enough to start the video transfer but the audio is stuttering horribly.  How're you able to get the source to compile for L4T?  Mine segfaults after building on the Nano when switching threads.

**Comment by gabrielformiga** [comment]
I used the scripts provided by @cgutman at "https://github.com/cgutman/moonlight-packaging" using my regular mac, it uses docker for building the package, so it's building the packages using my mac, I also had to modify the repository location at "build-package.sh" so it points to my repository "https://github.com/gabrielformiga/moonlight-qt.git" and finally modified "build-common.sh" so it stops trying to pull docker images at every build try.
Basically this is the build command for L4T Bionic "./build-l4t-bionic.sh YOUR_COMMIT_HASH" where mine for this bug fix is "937dce04513c68ea27350b9134f90c00961ca8fe"

**Comment by ToddAWalter** [comment]
Thanks!  I'm able to stream Cyberpunk 2077 to my phone w/o audio pops (over WiFi no less, and the prior build worked, so I'm pretty sure it's the L4T code.  The log shows 0.1% FPS drop but the first audio frame doesn't show up until 400ms into the stream so something goofy is going on there.

**Comment by hrfried** [comment]
Fwiw I just busted this out for the first time in a few months and upgraded the system/moonlight. Both v6 versions are broken for me (jetson nano dev kit) and I don't care to debug right now, but rolling back to the most recent v5 release started up no problem. Looks to be qml UI bugs/mismaps on moonlight side. Specifically hitting my sunshine host shows mismapped elements that I don't have energy to debug at this point. Might be other stuff going on but thats the main thing that jumped out to me.

For the record, if anyone is just wanting to use moonlight on their TV and think about upgrading later, it's simple to roll back with `sudo apt install moonlight-qt=5.0.1-1`

That's the most recent version that worked for me with no issues. You can see all the versions with `sudo apt show -a moonlight-qt`. 

And I thank the devs for having the prudence to keep all releases archived. :)

**Comment by gabrielformiga** [comment]
@hrfried I fixed the qml bugs that was preventing opening the stream, did you try it?

**Comment by hrfried** [comment]
@gabrielformiga I didn't as I'm sick and was a bit too tired to vet it or think about it too much. But I'll give it a go once I have some free time/brain. :)

**Comment by bietiekay** [comment]
came here to say I have the same problem with 6.0.1 on Rpi 4

**Comment by cgutman** [comment]
@bietiekay which OS version are you running on your Pi? It's working locally for me on Raspberry Pi OS Bookworm aarch64.

**Comment by bietiekay** [comment]
Linux raspberrypi 5.10.103-v7l+ #1529 SMP Tue Mar 8 12:24:00 GMT 2022 armv7l GNU/Linux

**Comment by gabrielformiga** [comment]
@bietiekay, try building from my branch, I fixed that bug, there's currently a opened PR with that fix, but not merged yet

**Comment by cgutman** [comment]
> Linux raspberrypi 5.10.103-v7l+ #1529 SMP Tue Mar 8 12:24:00 GMT 2022 armv7l GNU/Linux

That's your Linux kernel version. I need the distro version (for example, from `lsb_release -a`).

**Comment by bietiekay** [comment]
pi@raspberrypi:~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
